stdlib/public/runtime/ImageInspectionCommon.cpp

```diff
@@ -173,16 +159,35 @@ const swift::MetadataSections *swift_getMetadataSection(size_t index) {
 }
 
 SWIFT_RUNTIME_EXPORT
-const char *swift_getMetadataSectionName(void *metadata_section) {
+const char *
+swift_getMetadataSectionName(const swift::MetadataSections *section) {
   swift::SymbolInfo info;
-  if (lookupSymbol(metadata_section, &info)) {
+  if (lookupSymbol(section, &info)) {
     if (info.fileName) {
       return info.fileName;
     }
   }
   return "";
 }
 
+SWIFT_RUNTIME_EXPORT
+void swift_getMetadataSectionBaseAddress(const swift::MetadataSections *section,
+                                         void const **out_actual,
+                                         void const **out_expected) {
+  *out_actual = nullptr;
+  *out_expected = section->baseAddress;
+
+  swift::SymbolInfo info;
+  if (lookupSymbol(section, &info)) {
+    *out_actual = info.baseAddress;
+    if (info.fileName) {
+      return info.fileName;
```

Comment:
This causes linux bots to fail e.g. here: https://ci.swift.org/job/oss-swift-incremental-RA-linux-ubuntu-16_04/18176/

Reply:
I have reverted this commit. Please re-commit with a fix.

https://github.com/apple/swift/pull/40656

Reply:
Oh how odd. Clearly wrong—must be a merge error I missed that didn't get caught during testing. Can push a fix.